health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/ViewDashboard.js

```
@@ -0,0 +1,194 @@
+
+const ViewDashbaord = ({ stateCode }) => {
+  const { t } = useTranslation();
+  const location = useLocation();
+  const history = useHistory();
+  const tenantId = Digit?.ULBService?.getCurrentTenantId();
+  const project = location?.state?.project;
+  const campaignId = project?.referenceID;
+  const [redirected, setRedirected] = useState(false);
+  const queryStrings = Digit.Hooks.useQueryParams();
+
+  const { isLoading: campaignSearchLoading, data: campaignData, error: campaignError, refetch: refetch } = Digit.Hooks.campaign.useSearchCampaign({
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)